en_US.json

```diff
@@ -114,7 +114,9 @@
 	"cacertificate.create.button":"Create CA Certificate",
 	"cacertificate.create.title":"Create a CA Certificate",
 	"cacertificate.edit.title":"CA Certificate",
+	"cacertificate.error.chain.certificate.empty":"The chain certificate cannot be empty.",
```

Comment:
As commented on Jira https://quantil.atlassian.net/browse/NCP-3762?focusedCommentId=1507930, we should use "required" as the error message on the CA certificate creation page, if Chain Certificate is not specified.

Reply:
Understood. Have updated the implementation accordingly.